services/brig/test/integration/API/User.hs

```diff
@@ -28,9 +28,9 @@ tests conf p b c ch g aws = do
     at <- optOrEnv (Opt.setActivationTimeout . Opt.optSettings)                    conf read                     "USER_ACTIVATION_TIMEOUT"
     z  <- mkZAuthEnv conf
     return $ testGroup "user"
-        [ API.User.Account.tests       cl at conf p b c ch g aws
+        [ API.User.Client.tests        cl at conf p b c g
```

Comment:
I kinda just re-arrange tests so mine run first because it's easiest. If people actually care I can switch it back 🤷‍♂

Reply:
don't care, assuming you're not switching to avoid effect leakage between some of the lines.